tests/unit/src/test/resources/definition-scala3/example/Scalalib.scala

```diff
@@ -0,0 +1,26 @@
+package example
+
+class Scalalib/*Scalalib.scala*/ {
+  val nil/*Scalalib.scala*/ = List/*package.scala*/()
+  val lst/*Scalalib.scala*/ = List/*package.scala*/[
+    (
+        Nothing,
```

Comment:
Very similar as for Scala 2, just for `==` we don't have def, but this doesn't seem ncessary.